pytest_mh/utils/selinux.py

```diff
@@ -0,0 +1,102 @@
+"""SELinux utilities."""
+
+from __future__ import annotations
+
+import re
+
+from .. import MultihostHost, MultihostUtility
+from ..conn import ProcessLogLevel
+
+__all__ = [
+    "SELinuxContext",
+    "SELinux",
+]
+
+
+class SELinuxContext(object):
+    """
+    Result of ``ls -Z``
+    """
+
+    def __init__(self, user: str, role: str, type: str, sensitivity: str, categories: str | None) -> None:
+        self.user: str = user
+        """
+        SELinux user.
+        """
+
+        self.role: str = role
+        """
+        SELinux role.
+        """
+
+        self.type: str = type
+        """
+        SELinux type.
+        """
+
+        self.sensitivity: str = sensitivity
+        """
+        SELinux sensitivity level.
+        """
+
+        self.categories: str | None = categories
+        """
+        SELinux categories.
+        """
+
+    def __str__(self) -> str:
+        return f"({self.user}:{self.role}:{self.type}:{self.sensitivity}:{self.categories})"
+
+    def __repr__(self) -> str:
+        return str(self)
+
+    def __eq__(self, other) -> bool:
+        if not isinstance(other, SELinuxContext):
+            raise TypeError(f"Unexpected type: {type(other)}, expecting SELinuxContext")
```

Comment:
Comparing with other type means not equal.
If I want to compare given context for example to None this way it will work.
```suggestion
        if not isinstance(other, SELinuxContext):
            return False
```

Reply:
Makes sense. Updated.